webapp/js/app.js

```diff
@@ -27,6 +27,7 @@ var ModelList = require('./components/ModelList');
 var ModelEdit = require('./components/ModelEdit');
 var KeypairAdd = require('./components/KeypairAdd');
 var SigningLog = require('./components/SigningLog');
+import AccountList from './components/AccountList';
```

Comment:
Yes, the new-style components need an import. We can switch things over to import completely. It will take a bit more work to bring everything to ES6. By then, we'll probably find that ES7 has changed all the rules again!